Missing dependency "chalk"

Aarrghh... lesson in testing here.  I just tried installing the latest version on a system that had not had this plugin before.  We are missing a dependency...

```
[11/7/2023, 11:51:42 AM] [Homebridge UI] [homebridge-midea-platform] Child process ended
[11/7/2023, 11:51:49 AM] [Homebridge UI] [homebridge-midea-platform] Terminating child process...
[11/7/2023, 11:52:07 AM] [Homebridge UI] [homebridge-midea-platform] node:internal/modules/cjs/loader:1078
  throw err;
  ^

Error: Cannot find module 'chalk'
Require stack:
- /var/lib/homebridge/node_modules/homebridge-midea-platform/homebridge-ui/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/var/lib/homebridge/node_modules/homebridge-midea-platform/homebridge-ui/server.js:18:15)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/var/lib/homebridge/node_modules/homebridge-midea-platform/homebridge-ui/server.js'
  ]
}

Node.js v18.16.0
[11/7/2023, 11:52:07 AM] [Homebridge UI] [homebridge-midea-platform] Child process ended
```

--- a/homebridge-ui/server.js
+++ b/homebridge-ui/server.js
@@ -15,8 +15,6 @@ const { LocalSecurity } = require("../dist/core/MideaSecurity.js");
 const { PromiseSocket } = require("../dist/core/MideaUtils.js");
 const { defaultConfig, defaultDeviceConfig } = require('../dist/platformUtils.js');
 
-const chalk = require("chalk");
-
 var _ = require('lodash');
 
 /*********************************************************************
@@ -25,22 +23,35 @@ var _ = require('lodash');
  */
 class Logger {
   _debug;
+  _Reset = "\x1b[0m";
+  _Bright = "\x1b[1m";
+  _Dim = "\x1b[2m";
+
+  _FgBlack = "\x1b[30m";
+  _FgRed = "\x1b[31m";
+  _FgGreen = "\x1b[32m";
+  _FgYellow = "\x1b[33m";
+  _FgBlue = "\x1b[34m";
+  _FgMagenta = "\x1b[35m";
+  _FgCyan = "\x1b[36m";
+  _FgWhite = "\x1b[37m";
+  _FgGray = "\x1b[90m";
+
   constructor(uiDebug = false) {
-    chalk.level = 1;
     this._debug = uiDebug;
   }
   info(str) {
-    console.info(chalk.white(str));
+    console.info(this._FgWhite + str + this._Reset);
   }
   warn(str) {
-    console.warn(chalk.yellow(str));
+    console.warn(this._FgYellow + str + this._Reset);
   }
   error(str) {
-    console.error(chalk.red(str));
+    console.error(this._FgRed + str + this._Reset);
   }
   debug(str) {
     if (this._debug) {
-      console.debug(chalk.gray(str));
+      console.debug(this._FgGray + str + this._Reset);
     }
   }
   setDebugEnabled(enabled = true) {

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "displayName": "Homebridge Midea",
   "name": "homebridge-midea-platform",
-  "version": "0.4.1",
+  "version": "0.4.2",
   "description": "Homebridge plugin for Midea devices",
   "license": "Apache-2.0",
   "repository": {
@@ -30,7 +30,6 @@
   "dependencies": {
     "@homebridge/plugin-ui-utils": "^1.0.0",
     "axios": "^1.5.0",
-    "chalk": "^5.3.0",
     "fast-xml-parser": "^4.2.7",
     "lodash": "^4.17.21",
     "luxon": "^3.4.2",

